student-information-management-application/src/main/java/com/student/information/management/add/app/model/student/Student.java

```diff
@@ -0,0 +1,204 @@
+package com.student.information.management.add.app.model.student;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * This is the Student Model.
+ */
+public class Student {
```

Comment:
keep the POJO clean. remove unnecessary methods

Reply:
Deleted na po.